src/c++/perf_analyzer/client_backend/triton/triton_client_backend.cc

```diff
@@ -590,6 +590,19 @@ TritonClientBackend::ParseInferOptionsToTriton(
   }
   triton_options->triton_enable_empty_final_response_ =
       options.triton_enable_empty_final_response_;
+
+  for (auto& map_entry : options.request_parameters_) {
+    auto rp = tc::RequestParameter();
+    rp.str_value = map_entry.second.str_value;
+    rp.int_value = map_entry.second.int_value;
+    rp.bool_value = map_entry.second.bool_value;
+    uint64_t val =
+        static_cast<std::underlying_type<cb::RequestParameterType>::type>(
+            map_entry.second.type);
+    rp.type = static_cast<tc::RequestParameterType>(val);
+    triton_options->request_parameters.insert(
```

Comment:
Can't you just do the usual `triton_options->request_parameters[map_entry.first] = rp;`

Reply:
Updated since your suggestion is cleaner.